.gitignore

```diff
@@ -1,7 +1,7 @@
 .php_cs.cache
 composer.phar
 composer.lock
+phpunit.xml
```

Comment:
I've misread. I was think you were adding a file called `phpunit.xml`. It's in perfectly ok in `.gitignore`. Sorry.